libs/checkout/src/order/effects/order.effects.spec.ts

```diff
@@ -4,19 +4,22 @@ import { Observable, of } from 'rxjs';
 import { hot, cold } from 'jasmine-marbles';
 
 import { DaffCart, DaffCartDriver, DaffCartServiceInterface } from '@daffodil/cart';
-import { DaffCartFactory, DaffTestingCartService } from '@daffodil/cart/testing';
-import { DaffTestingCheckoutService } from '@daffodil/checkout/testing';
+import { DaffTestingCartService, DaffCartFactory } from '@daffodil/cart/testing';
+import { DaffTestingCheckoutService, DaffOrderFactory } from '@daffodil/checkout/testing';
 
 import { OrderEffects } from './order.effects';
 import { DaffPlaceOrder, DaffPlaceOrderSuccess, DaffPlaceOrderFailure } from '../actions/order.actions';
 import { DaffCheckoutServiceInterface } from '../../drivers/interfaces/checkout-service.interface';
 import { DaffCheckoutDriver } from '../../drivers/injection-tokens/driver-checkout.token';
+import { Order } from '@daffodil/checkout';
```

Comment:
Shouldn't this give you a circular dependency?

Reply:
Yes, it should. But, since we do a build before we run tests, it doesn't since the token is available from the package.

Reply:
Fixed in 3faccf704b20df758896152ff5f4b1c21b7012db